ptf_tests/p4ovs_test_runner.py

```diff
@@ -149,12 +149,21 @@ def print_help(self):
             if i.startswith("#"):
                 continue
             items = i.strip().split(":")
-            test_to_run[items[0].strip()] = ':'.join(items[1:]).strip()
-            sequence.append(items[0])
+
+            # if there are tests with same script name but different parameters (json file etc.)
+            if scriptname in test_to_run.keys():
+                scriptname = scriptname + "_2"
```

Comment:
why this is "_2"? shouldn't this be generic to cover multiple?

Reply:
Fixed